lib/widget/search/main_search_bar.dart

```diff
@@ -328,7 +328,7 @@ class PositionedFloatingSearchBar extends SearchDelegate<String> {
   }
 
   // Google places API request for Place Details
-  Future<Coordinate> _getPlaceDetails(String placeId) async {
+  Future<Coordinate> getPlaceDetails(String placeId) async {
```

Comment:
Just a general warning to state that the access modifier was changed from private to public to be used in a method handleMapOnLongPress in Map. Map is our God class that depends on everything, and I'm honestly not sure how we should go about possibly refactoring this without breaking things in Flutter? I See why this is required for the long press feature in this PR, but we should try our best to find solutions to avoid accumulating more debt then we already have in Map. I have no issues with approving this PR as is, but if anyone in the team can come up with any ideas, they would be more than welcome